src/engine.ts

```diff
@@ -331,4 +384,15 @@ export default class Engine {
     this.players[player].pass();
     (this[Command.ChooseRoundBooster] as any)(player, booster, Command.Pass);
   }
+
+  [Command.Leech](player: PlayerEnum, leech: number) {
+    const powerLeeched = this.players[player].data.chargePower(leech);
```

Comment:
You need to check the leech is the same as in the available command

Reply:
ok, done